frontend/app/routes/$lang/_public/status/$id/result.tsx

```diff
@@ -0,0 +1,72 @@
+import { json } from '@remix-run/node';
+import type { LoaderFunctionArgs } from '@remix-run/node';
+import { useLoaderData, useParams } from '@remix-run/react';
+
+import { faChevronRight } from '@fortawesome/free-solid-svg-icons';
+import { useTranslation } from 'react-i18next';
+
+import pageIds from '../../../page-ids.json';
+import { ButtonLink } from '~/components/buttons';
+import { ClientFriendlyStatusMarkdown } from '~/components/client-friendly-status-markdown';
+import { ContextualAlert } from '~/components/contextual-alert';
+import { InlineLink } from '~/components/inline-link';
+import { useFeature } from '~/root';
+import { loadStatusState } from '~/route-helpers/status-route-helpers.server';
+import { featureEnabled, getEnv } from '~/utils/env-utils.server';
+import { getTypedI18nNamespaces } from '~/utils/locale-utils';
+import { getFixedT } from '~/utils/locale-utils.server';
+import type { RouteHandleData } from '~/utils/route-utils';
+
+export const handle = {
+  i18nNamespaces: getTypedI18nNamespaces('status', 'gcweb'),
+  pageIdentifier: pageIds.public.status.myself,
+  pageTitleI18nKey: 'status:myself.page-title',
+} as const satisfies RouteHandleData;
+
+export async function loader({ context: { session }, params, request }: LoaderFunctionArgs) {
+  featureEnabled('status');
+  const { ENABLED_FEATURES, HCAPTCHA_SITE_KEY } = getEnv();
+  const { statusCheckResult } = loadStatusState({ params, session });
+
+  const csrfToken = String(session.get('csrfToken'));
+  const t = await getFixedT(request, handle.i18nNamespaces);
+
+  const hCaptchaEnabled = ENABLED_FEATURES.includes('hcaptcha');
+
+  const meta = { title: t('gcweb:meta.title.template', { title: t('status:myself.page-title') }) };
+
+  return json({ statusResult: statusCheckResult, csrfToken, hCaptchaEnabled, meta, siteKey: HCAPTCHA_SITE_KEY });
```

Comment:
Loader data such as `csrfToken`, `hCaptchaEnabled`, and `siteKey` aren't being used in the `StatusCheckerResult` component and can be removed.

Reply:
Removed unused loader params (not sure if hcaptcha is also required on myself/child pages)

Reply:
hCaptcha should be on the form before the final submission step, which in the case of status are the myself/child pages.